impala/datadog_checks/impala/metrics_daemon.py

```diff
@@ -0,0 +1,388 @@
+                "type": "monotonic_count",
+            },
+            "write_size": {
+                "new_name": "daemon.io_mgr_queue.write_size.quantile",
+                "type": "gauge",
+            },
+            "write_size_sum": {
+                "new_name": "daemon.io_mgr_queue.write_size.sum",
+                "type": "monotonic_count",
+            },
+            "write_size_count": {
+                "new_name": "daemon.io_mgr_queue.write_size.count",
+                "type": "monotonic_count",
+            },
+        },
+    },
+    "^impala_buffer_pool_arena_([0-9]+)_(.+)$": {
+        "label_name": "id",
+        "sub_metrics": {
+            "num_final_scavenges_total": {
+                "new_name": "daemon.buffer_pool.arena.num_final_scavenges_total.count",
+                "type": "monotonic_count",
+            },
+            "num_scavenges_total": {
+                "new_name": "daemon.buffer_pool.arena.num_scavenges_total.count",
+                "type": "monotonic_count",
+            },
+            "clean_page_hits_total": {
+                "new_name": "daemon.buffer_pool.arena.clean_page_hits_total.count",
+                "type": "monotonic_count",
+            },
+            "system_alloc_time_total": {
+                "new_name": "daemon.buffer_pool.arena.system_alloc_time_total.count",
+                "type": "monotonic_count",
+            },
+            "local_arena_free_buffer_hits_total": {
+                "new_name": "daemon.buffer_pool.arena.local_arena_free_buffer_hits_total.count",
+                "type": "monotonic_count",
+            },
+            "numa_arena_free_buffer_hits_total": {
+                "new_name": "daemon.buffer_pool.arena.numa_arena_free_buffer_hits_total.count",
+                "type": "monotonic_count",
+            },
+            "direct_alloc_count_total": {
+                "new_name": "daemon.buffer_pool.arena.direct_alloc_count_total.count",
+                "type": "monotonic_count",
+            },
+            "allocated_buffer_sizes": {
+                "new_name": "daemon.buffer_pool.arena.allocated_buffer_sizes.quantile",
+                "type": "gauge",
+            },
+            "allocated_buffer_sizes_sum": {
+                "new_name": "daemon.buffer_pool.arena.allocated_buffer_sizes.sum",
+                "type": "monotonic_count",
+            },
+            "allocated_buffer_sizes_count": {
+                "new_name": "daemon.buffer_pool.arena.allocated_buffer_sizes.count",
+                "type": "monotonic_count",
+            },
+        },
+    },
+    "^impala_rpc_impala_(.+Service)_(.+)$": {
+        "label_name": "daemon_service",
+        "sub_metrics": {
+            "rpcs_queue_overflow_total": {
+                "new_name": "daemon.rpcs_queue_overflow.count",
+                "type": "monotonic_count",
+            },
+        },
+    },
+    "^impala_mem_tracker_(.+Service)_(.+)$": {
```

Comment:
Could we add more test metrics like this but with more complex service names such as "test_service" or "test.service"

Reply:
Not sure to understand what you mean. At the moment we have only two variants for this metric: `DataStreamService` and `ControlService`. Do you thing we should update the regex we have here?

Reply:
Mhm so are `DataStreamService` and `ControlService` the only two possible options? I was wondering if a user could name their service `Control-Service` or something that could potentially break a regex

Reply:
users do not have the control on this part, this `service` is not what we (Datadog) call a service, it's an internal service used only by the impala deamon